ingestion/functions/template.yaml

```diff
@@ -194,6 +194,30 @@ Resources:
         - AWSLambdaReadOnlyAccess
       Layers:
         - !Ref ParsingLibLayer
+  BrazilParaibaParsingFunction:
```

Comment:
why is Paraiba parser in this PR?

Reply:
this is just the most up-to-date template.yaml file pulled from main, with argentina added at bottom. Should it not be there?

Reply:
seems to have been included twice, probably my fault, should be fixed in latest commit